controllers/drcluster_controller.go

```diff
@@ -141,6 +145,24 @@ func (r *DRClusterReconciler) drClusterConfigMapMapFunc(configMap client.Object)
 	return requests
 }
 
+func (r *DRClusterReconciler) drClusterSecretMapFunc(secret client.Object) []reconcile.Request {
+	if secret.GetNamespace() != NamespaceName() {
+		return []reconcile.Request{}
+	}
+
+	drcusters := &ramen.DRClusterList{}
+	if err := r.Client.List(context.TODO(), drcusters); err != nil {
+		return []reconcile.Request{}
+	}
+
+	requests := make([]reconcile.Request, len(drcusters.Items))
+	for i, drcluster := range drcusters.Items {
+		requests[i].Name = drcluster.GetName()
```

Comment:
Doesn't this loop copy all drclusters from drcusters.Items? We can avoid the copy if we do:

    for i := range drcusters.Items {
        requests[i].Name = drclusters.Items[i].GetName()
    }

Reply:
Done